### DISCUSSION TITLE
No language identified

### DISCUSSION THREAD

**Discussion by Leni-Nikitaa** [github-linguist/linguist#6458] [Classification] [answered]
Hi I am new to GitHub.
We have made a fitness tracker in our repository:
[Link to repository: Fitness-Tracker](https://github.com/Revan-Bairav/Fitness-Tracker)
GitHub is not displaying any language for my repository, please help
@lildude I've seen you helping with issues could you please offer some help

**Comment by lildude** [comment, marked as answer]
Your repo has no code in the default branch (`main`) so there's nothing to identify 😁.

From the first line of [How Linguist works on GitHub.com](https://github.com/github-linguist/linguist/blob/master/docs/how-linguist-works.md#how-linguist-works-on-githubcom):

> When you push changes to a repository on GitHub.com, a low priority background job is enqueued to analyze the [default branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches#about-the-default-branch) of your repository as explained above.

So if you want language analysis performed, you need to move your code to your default branch or change your default branch from `main` to `master` (as I see all your code there).

**Reply by Leni-Nikitaa** [reply]
I've forked the repository and changed the `master` to default but still no change. Should it be done in the original repository?
Here's my [forked repository](https://github.com/Leni-Nikitaa/Fitness-Tracker)

**Reply by lildude** [reply]
Errrm, you marked everything as `vendored` in your `.gitattributes` file so you've told Linguist to ignore everything in the repo.

Remove the line from your `.gitattributes` and all will be revealed as you can see in [my fork of your repo](https://github.com/lildude/Fitness-Tracker/tree/master) in which I've commented out that line:

![CleanShot 2023-06-19 at 16 52 03](https://github.com/github-linguist/linguist/assets/627280/40b6e323-ac06-41b7-a9a2-7c964015373d)

**Reply by Leni-Nikitaa** [reply]
Thank you so much!
